### ISSUE TITLE
Add Viper prefix for env var

### ISSUE THREAD

**Issue by fmorency** [liftedinit/manifest-ledger#57]
Current code is 

```
WithViper("") // In danger, we don't use any prefix for env variables.
```

which means that things like

```shell
HOME=$HOME/.poppy ./build/manifestd init baba
➜  manifest-ledger git:(f09ff6e) ✗ ls ~/.poppy 
config  data
```

"kinda" of work...

**Comment by fmorency** [comment]
The environment variable and Viper prefix are broken on 0.50.x.

https://github.com/cosmos/cosmos-sdk/pull/18345

**Comment by Reecepbcups** [comment]
`MANIFESTD_` works with viper :) sdk handles this under the hood for you already

proof
```
MANIFESTD_NODE=https://bad-rpc.com:443 manifestd status

...

post failed: Post "https://bad-rpc.com:443": dial tcp: lookup bad-rpc.com on 216.165.129.158:53: no such host
```

thus it used the manifest node specific

**Comment by Reecepbcups** [comment]
Closing since this is supported. re-tag me in this if this is not what you were referencing

**Comment by fmorency** [comment]
Hey @Reecepbcups, can you try

```shell
MANIFESTD_HOME=$HOME/.foofoo manifestd init foo
```

It doesn't work properly here on 0.50.x (was fixed current SDK main)

**Comment by Reecepbcups** [comment]
Oh I see, specifically it is the home env var breaking.

Since manifest uses a fork anyways we can backport to our fork. Is it a big enough issue we should?

Let me know if you want me too

**Comment by fmorency** [comment]
> Oh I see, specifically it is the home env var breaking.
> 
> Since manifest uses a fork anyways we can backport to our fork. Is it a big enough issue we should?
> 
> Let me know if you want me too

Non-blocking. We can wait for 0.51.x as we have a workaround (`--home`)

Thanks for offering @Reecepbcups, I appreciate it.
